bank/api/security_handlers.go

```diff
@@ -100,10 +103,10 @@ func LogoutHandler(sessionStore domain.SessionStore) http.Handler {
 			return
 		}
 
-		//json.NewEncoder(w).Encode(session)
-		*cookie = http.Cookie{Name: "SID", Path: "/", Value: "deleted", MaxAge: -1}
-		http.SetCookie(w, cookie)
+		//*cookie = http.Cookie{Name: "SID", Path: "/", Value: "deleted", MaxAge: -1}
```

Comment:
Logout handler doesn't have to return session as it will be removed with the request. 

You could just return http.StatusOK to indicate that session was removed successfully and the UI have to remove this cookie.